src/Makefile

```
@@ -149,16 +149,15 @@ endif
 
 CLANG_LDFLAGS := $(LLVM_LDFLAGS)
 ifeq ($(OS), Darwin)
-CLANG_LDFLAGS += -Wl,-undefined,dynamic_lookup
-OSLIBS += $(SRCDIR)/mach_dyld_atfork.tbd
+CLANG_LDFLAGS += -Wl,-undefined,dynamic_lookup -Wl,-U,__dyld_atfork_parent -Wl,-U,__dyld_atfork_prepare -Wl,-U,__dyld_dlopen_atfork_parent -Wl,-U,__dyld_dlopen_atfork_prepare -framework CoreFoundation
```

I don't see how this is clang related, or why you are adding the clang flags to all compilers.

```suggestion
OSLIBS += -Wl,-undefined,dynamic_lookup -Wl,-U,__dyld_atfork_parent -Wl,-U,__dyld_atfork_prepare -Wl,-U,__dyld_dlopen_atfork_parent -Wl,-U,__dyld_dlopen_atfork_prepare -framework CoreFoundation
```

---

Agreed. I moved the flags to `OSLIBS`.